jetty-server/src/main/config/etc/jetty-connectionlimit.xml

```diff
@@ -1,13 +1,17 @@
 <?xml version="1.0"?>
 <!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "http://www.eclipse.org/jetty/configure_9_3.dtd">
-
 <Configure id="Server" class="org.eclipse.jetty.server.Server">
   <Call name="addBean">
     <Arg>
       <New class="org.eclipse.jetty.server.ConnectionLimit">
-        <Arg type="int"><Property name="jetty.connection.limit" default="1000"/></Arg>
-	<Arg><Ref refid="Server"/></Arg>
+        <Arg name= "maxConnections" type="int">
+          <Property name="jetty.connection.limit" default="1000" />
```

Comment:
Same comments as above.

Module should be named `connectionlimiter`, and properties should be `jetty.connectionlimiter.maxConnections` and `jetty.connectionlimiter.idleTimeout`.

Reply:
The problem here is that `connectionlimit` is an existing class/module/xml etc., so we can't really change the name, unless we provide deprecated versions.   The accept rate naming followed this naming pattern.....  so I guess deprecated and going with the better names is best????